src/frontend/shortlist/src/components/preferences/PreferenceDemo.vue

```diff
@@ -44,6 +47,36 @@ export default {
   },
   setup() {
     return {
+      question1: {
+        Name: "Instruction",
+        Text: "How much do you value instructions from your teachers?",
+        Min: 1,
+        MinLabel: "I'd like more time studying alone.",
```

Comment:
```suggestion
        MinLabel: "Less Important",
```

Reply:
Hey this one I think mine's better, it relates XD

Reply:
Maybe just, "I'm a loner type scholar"